CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Changelog for NeoFS Node
 - Failure of 1st split-chain child processing during metabase migration (#3187)
 - Retries to sending notary request (#3193)
 - Initial epoch tick by a just-started IR (#3007)
+- Retries to `Invoke` function (#3200)
```

Comment:
No one can understand what this means, to users it's more like "Missing retries when sending transaction/notary request leading to request failure in case of temporary errors". Which can be unified with the "notary request" two lines above.